src/Dfc.CourseDirectory.Core/ReferenceData/Lars/LarsDataImporter.cs

```diff
@@ -187,21 +200,31 @@ Task ImportCategoryToSql()
 
             Task ImportLearnAimRefTypeToSql()
             {
-                var records = ReadCsv<UpsertLarsLearnAimRefTypesRecord>("LearnAimRefType.csv");
+                const string csv = "LearnAimRefType.csv";
+                var records = ReadCsv<UpsertLarsLearnAimRefTypesRecord>(csv).ToList();
+
+                static bool Excluded(UpsertLarsLearnAimRefTypesRecord r) => r.LearnAimRefTypeDesc.StartsWith("T Level", StringComparison.InvariantCultureIgnoreCase);
+                var excluded = records.Where(Excluded).Select(r => r.LearnAimRefType);
+                Console.Out.WriteLine($"{csv} - Excluded {nameof(UpsertLarsLearnAimRefTypesRecord.LearnAimRefType)}s: {string.Join(",", excluded)} (T Level detected in {nameof(UpsertLarsLearnAimRefTypesRecord.LearnAimRefTypeDesc)})");
```

Comment:
Use ILogger here instead of going direct to Console

Reply:
done, we'll have a look tomorrow to see if we want to iterate further :+1: